### DISCUSSION TITLE
Making Bot to turn off itself when it is in the specified coords

### DISCUSSION THREAD

**Discussion by KatelynnChan** [PrismarineJS/mineflayer#2279] [Q&A]
Making Bot to turn off itself when it is in the specified coords, I want to specify the coords in the index.js I dont need it to be a chat command. When it reaches the coords or at least the x or the z coords the bot will leave the server. I would also like the bot to send its coords to the console every now and then.

**Comment by amoraschi** [comment]
You can check if the `bot.entity`'s `position.x` and `position.z` are equal to the specified `x` and `z`, and if it returns true, do `bot.quit`. Alternatively you can check if the bot's position is close to the desired position by using **Vec3**`.distanceTo`(**Vec3**).

You can make an Interval that keeps printing the bot's position.
